udfs/community/gunzip.sqlx

```diff
@@ -45,7 +45,7 @@ if (gzipped == null) {
   return null;
 }
 
-const bytes = base64_js.toUint8Array(gzipped);
+const bytes = base64_js.toByteArray(gzipped);
```

Comment:
Looks like I wanted to add [`js-base64`](https://www.npmjs.com/package/js-base64), not [`base64-js`](https://www.npmjs.com/package/base64-js) 🤦 

Both seem heavily used, so I guess either one will do...

Reply:
GitHub repo for base64-js shows >20M  "Used by" while js-base64 shows ~5M "Used by".
I haven't benchmarked them against each other so just going by popularity, we can keep base64-js. 
Feel free to benchmark them and pick the fastest performing one.